src/benchmark_api_internal.cc

```diff
@@ -50,6 +51,11 @@ BenchmarkInstance::BenchmarkInstance(Benchmark* benchmark, int family_idx,
     name_.min_time = StrFormat("min_time:%0.3f", benchmark_.min_time_);
   }
 
+  if (!IsZero(benchmark->min_warmup_time_)) {
+    name_.min_warmup_time =
+        StrFormat("min_warmup_time:%0.3f", benchmark_.min_warmup_time_);
```

Comment:
why is it important to have this in the name?

Reply:
I don't really think it is important. However I would assume its the same reasoning as for having min_time in the name of the benchmark which I assume is:
Something might change on the results depending on this value. In this case this might be figuring out how long to run this benchmark on a certain platform until I am sure the caches are warmed up which might be easier if I can register the benchmark a few times with different warm up phases to see where the results saturate. If you think it shouldn't be in the name I am happy to remove it :)

Reply:
it's fine :)